src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -113,6 +115,32 @@ function isPreview() {
     return $(document).find('.navbar').text().indexOf('Preview') !== -1;
 }
 
+function isModeration() {
+    return $('#moderationHintButton').length !== 0;
+}
+
+function bindModerationHintButton() {
+    if (!isModeration()) {
```

Comment:
I was just wondering, could we simply do the following? 🤔 
```
if (!isModeration()) {
    var expandText = '[Show more]';
    ...
```
To me that seems more straightforward, as it tells devs that the method body will be executed only if user is currently moderating